<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>274</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>&amp;Fichier</string>
    </property>
    <widget class="QMenu" name="menu_Fichiers_r_cents">
     <property name="title">
      <string>&amp;Fichiers r√©cents</string>
     </property>
     <addaction name="actionfichiers1"/>
     <addaction name="actionfichier2"/>
     <addaction name="actionfichier3"/>
     <addaction name="separator"/>
    </widget>
    <addaction name="action_Ouvrir_fichier"/>
    <addaction name="menu_Fichiers_r_cents"/>
    <addaction name="action_Save"/>
    <addaction name="action_Quitter"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_Aide">
    <property name="title">
     <string>&amp;Aide</string>
    </property>
    <addaction name="action_A_propos"/>
   </widget>
   <widget class="QMenu" name="menuO_utils">
    <property name="title">
     <string>O&amp;utils</string>
    </property>
    <addaction name="action_Configuration"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuO_utils"/>
   <addaction name="menu_Aide"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Ouvrir_fichier">
   <property name="text">
    <string>&amp;Ouvrir fichier</string>
   </property>
  </action>
  <action name="action_Quitter">
   <property name="text">
    <string>&amp;Quitter</string>
   </property>
  </action>
  <action name="actionfichiers1">
   <property name="text">
    <string>fichiers1</string>
   </property>
  </action>
  <action name="actionfichier2">
   <property name="text">
    <string>fichier2</string>
   </property>
  </action>
  <action name="actionfichier3">
   <property name="text">
    <string>fichier3</string>
   </property>
  </action>
  <action name="action_A_propos">
   <property name="text">
    <string>&amp;A propos</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="action_Configuration">
   <property name="text">
    <string>&amp;Configuration</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
